### PR TITLE
Arabic translation

### PR DESCRIPTION
**Pull request by EZ0010**
The Arabic translation of Phoenix Wallet, I tried to make it as clear as possible.

I invited some people on Twitter to review this translation and fix what is wrong. [tweet link](https://twitter.com/Bitcoin_info_ar/status/1420780148811575296)

### CODE REVIEW

File: app/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -16,739 +16,689 @@
...
-    <p>Payment channels will be automatically created for you if needed.</p>
-    <p>This feature is required for the background payment reception to work properly, but you can disable it if you want (in Settings > Payment settings).</p>
-    <p>The channel creation cost still applies (%1$s%%).</p>
-
-    <h4>Improved wallet lock</h4>
-    <p>The app lock now uses the system\'s lock. No need to remember yet another PIN!</p>
-    <p>Go to Settings > Security to enroll for the new screen lock feature.</p>
-  ]]></string>
+<h4> استقبال الدفع في الخلفية </h4>
+ <p> أصبح بإمكان Phoenix الآن تلقي المدفوعات حتى إذا لم يتم تشغيل التطبيق. </p>
```
Comment (Jean-Gray) [inline app/src/main/res/values-ar/strings.xml:684]: . الآن تلقي المدفوعات حتى إذا لم يتم تشغيل التطبيق Phoenix أصبح بإمكان

File: app/src/main/res/values-ar/strings.xml [hunk 2]
```diff
@@ -16,739 +16,689 @@
...
+ الرجاء معاودة المحاولة في وقت لاحق.
+]]></string>
   <string name="receive_swap_in_disabled"><![CDATA[
-    <b>Service disabled</b>
-    <br /><br />
-    The on-chain deposit swap service has been temporarily disabled.
-    <br /><br />
-    Try again later.
-  ]]></string>
+<b>تعطيل الخدمة</b>
```
Comment (Jean-Gray) [inline app/src/main/res/values-ar/strings.xml:125]: تم تعطيل الخدمه

File: app/src/main/res/values-ar/strings.xml [hunk 3]
```diff
@@ -16,739 +16,689 @@
...
+ <br /> <br />
+تم تعطيل خدمة مبادلة الايداعات على السلسلة الرئيسية مؤقتًا.
+ <br /> <br />
+ حاول مرة أخرى في وقت لاحق.
```
Comment (Jean-Gray) [inline app/src/main/res/values-ar/strings.xml:129]: كرر المحاوله لاحقاّ

File: app/src/main/res/values-ar/strings.xml [hunk 4]
```diff
@@ -16,739 +16,689 @@
...
+]]></string>
+  <string name="receive_swap_in_error_message">تعذر الحصول على عنوان بيتكوين من خدمة المبادلة.\n\nحاول مرة أخرى لاحقا.</string>
   <string name="receive_swap_in_unknown"><![CDATA[
-    <b>Unavailable</b>
-    <br /><br />
-    Please try again later.
-  ]]></string>
+<b>غير متوفر</b>
+ <br /> <br />
+ الرجاء معاودة المحاولة في وقت لاحق.
```
Comment (Jean-Gray) [inline app/src/main/res/values-ar/strings.xml:122]: برجاء تكرار المحاولة لاحقاّ

**Review by Jean-Gray — CHANGES_REQUESTED** [review]
Kindly check the suggested changes in the Arabic translation, Good luck
برجاء مراجعة التعديلات المقترحه في الترجمه العربيه ،، بالتوفيق

File: app/src/main/res/values-ar/strings.xml [hunk 5]
```diff
@@ -16,739 +16,689 @@
...
-    %1$s
-    <br /><br />
-    <a href="%2$s">Open Link</a>
-  ]]></string>
-  <string translatable="false" name="paymentdetails_lnurlpay_success_action_aes"><![CDATA[
-    %1$s
-    <br />
-    <b>%2$s</b>
-  ]]></string>
-  <string translatable="false" name="paymentdetails_lnurlpay_success_action_aes_failure">Message from service could not be decrypted.</string>
```
[the remark this answers is not in the document]
Comment (EZ0010) [reply app/src/main/res/values-ar/strings.xml]: this string is not translatable as described on the translatable="false"

**Comment by EZ0010** [comment]
> Kindly check the suggested changes in the Arabic translation, Good luck
> برجاء مراجعة التعديلات المقترحه في الترجمه العربيه ،، بالتوفيق

Thanks for your review, I edited most of your suggestions except the "Electrum" edit, I tend to not translate names of brands as it doesn't reflect anything when translated.

**Comment by EZ0010** [comment]
I built the app and tested it after adding the translation, everything worked well. no RTL issues. the app became RTL without doing anything.

Screen Shot 2021-07-31 at 5 59 01 PM


Screen Shot 2021-07-31 at 5 59 19 PM


Screen Shot 2021-07-31 at 5 59 38 PM

**Comment by dpad85** [comment]
Thanks a lot! 

There are some RTL glitches (like the back arrow pointing to the left and the currency picker being misaligned), but it's outside the scope of this PR. I'll review the translation asap.

**Comment by hima101** [comment]
if you mean the currency picker in the top of the screen that is because a lot of arabic countries uses Hindu–Arabic numeral system (١٢٣٤٥٦٧٨٩٠) not the Arabic numeral system (1234567890), and these Hindu–Arabic numbers are considered RTL letters that is why they look kinda weird next to "sats", I think the way to solve this is to translating "sats" or any currency into an arabic word. so it will look like (٤٠ ساتوشي) meaning (40 sats)

and btw it is also possible to make the language of the deivce Arabic with Arabic numeric system, just like this:-
Screen Shot 2021-08-10 at 10 14 30 AM

**Comment by EZ0010** [comment]
if you mean the currency picker in the top of the screen that is because a lot of arabic countries uses Hindu–Arabic numeral system (١٢٣٤٥٦٧٨٩٠) not the Arabic numeral system (1234567890), and these Hindu–Arabic numbers are considered RTL letters that is why they look kinda weird next to "sats", I think the way to solve this is by translating "sats" or any currency into an arabic word. so it will look like (٤٠ ساتوشي) meaning (40 sats)

and btw it is also possible to make the language of the deivce Arabic with Arabic numeric system, just like this:-
128824269-3cbc1fe8-6509-43f7-afd6-a65900da036b

**Review by dpad85 — APPROVED** [review]
LGTM

**Pull request merged**